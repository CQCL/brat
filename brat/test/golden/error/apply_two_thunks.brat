inc :: Int -> Int
inc(n) = succ(n)

thunks :: { Int -> Int }, {Int -> Int}
thunks = inc,{ x -> inc(inc(x)) }

# This obviously shouldn't work, but the error message could be much better!
go :: Int -> Int
go(n) = thunks(n)
