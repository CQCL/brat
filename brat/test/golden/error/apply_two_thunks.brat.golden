Error in test/golden/error/apply_two_thunks.brat@9:9
go(n) = thunks(n)
        ^^^^^^^^^

  Internal error: Expected empty leftover function args, got:
  (a1 :: Int)

